TouchEffect.Droid/PlatformTouchEff.cs

```diff
@@ -58,19 +58,75 @@ protected override void OnDetached()
             }
         }
 
+        bool _inrange;
+        bool _pressed;
+        bool _leftrange = false;
+        int[] twoIntArray = new int[2];
```

Comment:
please use _ also use "private" keyword
and choose a more suitable name for twoIntArray